config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-ci.yaml

```diff
@@ -1,65 +0,0 @@
-postsubmits:
```

Comment:
These jobs are not necessary anymore as we are using the postsubmit jobs which are pusing to staging